<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>420</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MessageLine">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TargetReceiver">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="messageListDisplay">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="retrybtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>retry</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
